enabler/build.gradle

```diff
@@ -10,7 +10,7 @@ android {
         ndk.abiFilters 'armeabi-v7a','arm64-v8a','x86','x86_64'
         applicationId "com.openxcplatform.enabler"
         minSdkVersion rootProject.ext.minSdkVersion
-        targetSdkVersion rootProject.ext.targetSdkVersion
+        targetSdkVersion 29
```

Comment:
I don't think you need to change this, if you already changed the root `build.gradle`? Mainly because then it becomes 2 places you need to make changes in the future.